src/components/Content.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Swal from 'sweetalert2'
+import AllToDos from "./AllToDos";
+import Form from "./Form";
+
+function Content(){
+
+    const [data, setData] = useState({
+        todos: [] as {id: number, title: string; completed: boolean }[],
+        isLoading: false,
+        error: ""
+    })
+
+    useEffect(()=>{
+        const fetchData= async ()=>{
+            try{
+                const todosList= (await axios.get("https://jsonplaceholder.typicode.com/todos")).data;
+                setData({...data, todos: todosList.slice(0, 10), isLoading: true}) // take just 10 elements instead of all the return from the API (200 elements)
+            }catch(e:any){
+                setData({...data, error:e, isLoading: true})
+            }
+        }
```

Comment:
this async-await will not work, since you are not awaiting the call of fetchData, also since you are calling it down-there this function shouldn't live in the useEffect!!